packages/union-component/src/machines/__tests__/membershipMachine.spec.ts

```diff
@@ -55,7 +55,10 @@ test('goes into process membership state after filling all information', () => {
 
   expect(machineState.context).toEqual({
     addressInformation,
-    personalInformation,
+    personalInformation: {
+      ...personalInformation,
+      phoneNumber: personalInformation.phoneNumber.replace(/ /g, '')
```

Comment:
@duranmla why we use different regex to replace the phone number?

Reply:
lets sync later about this. Also, if we are changing the phone input component, this code won't be used anymore